config/routes.rb

```diff
@@ -1583,6 +1583,16 @@ def patch(*)
 
     get "/theme-qunit" => "qunit#theme"
 
+    # This is a special route that is used when theme QUnit tests are run through testem which appends a testem_id to the
+    # path. Unfortunately, testem's proxy support does not allow us to easily remove this from the path, so we have to
+    # handle it here.
+    if Rails.env.development?
+      get "/testem-theme-qunit/:testem_id/theme-qunit" => "qunit#theme",
```

Comment:
Nice - I like the unique prefix - much better than my original suggestion of re-using `theme-qunit/` 🤩